node_modules/.yarn-integrity

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Should we exclude this?

Reply:
oh yes, accidentally installed in the root. will remove